detox/local-cli/templates/jest.js

```diff
@@ -1,19 +1,27 @@
 const firstTestContent = require('./firstTestContent');
 const runnerConfig = `{
     "setupFilesAfterEnv": ["./init.js"],
-    "testEnvironment": "node"
+    "testEnvironment": "node",
+    "reporters": ["detox/runners/jest/DetoxJestReporter.js"],
+    "verbose": true
 }
 `;
 
-const initjsContent = `const detox = require('detox');
-const config = require('../package.json').detox;
+const initjsContent = `const config = require('../package.json').detox;
```

Comment:
⚠️ @d4vidi, uh-oh. I think you accidentally removed:  ⚠️
`const detox = require('detox');`

The init code won't work correctly.

Reply:
My bad, it was an issue when I started reverting the adapter code.